Kudu.Services.Web/App_Start/NinjectServices.cs

```diff
@@ -628,6 +629,25 @@ private static void RemoveTempFileFromUserDrive(IEnvironment environment)
             FileSystemHelpers.DeleteDirectorySafe(Path.Combine(environment.RootPath, "data", "Temp"), ignoreErrors: true);
         }
 
+        // add/update gitconfig file to be equivalent of running
+        // git config --global --add safe.directory "*"
+        private static void EnsureGitConfigFileWithSafeDirectoryConfig(IEnvironment environment)
+        {
+            OperationManager.SafeExecute(() =>
+            {
+                var gitConfigPath = Path.Combine(environment.RootPath, ".gitconfig");
```

Comment:
we can move this after checking for marker file, as gitConfigPath won't be even used if marker file exist.

Reply:
updated, thanks!